app/Providers/AppServiceProvider.php

```diff
@@ -6,6 +6,19 @@
 
 class AppServiceProvider extends ServiceProvider
 {
+    /**
+     * Bootstrap any application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+		// Enfocre SSL in Production
```

Comment:
Use spaces for indentation
```suggestion
        // Enfocre SSL in Production
```